lib/private/strip-stack-message.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function stripStackMessage(e) {
```

Comment:
One other thing:  This is pretty complicated stuff, and I keep getting thrown off when I read it fresh.  It's pretty hard to think of a succinct name for this which really captures what it is...  But how about, in the interest of utter clarity, renaming this helper `getBareTraceFromError`?  That way, it's more obvious when you're looking at the userland code that this is something taking an Error instance and yielding a string.